### DISCUSSION TITLE
Java agent exporter issue.

### DISCUSSION THREAD

**Discussion by esaabdk** [open-telemetry/opentelemetry-java-instrumentation#10375] [Q&A]
Hi,

We are facing an issue with java agent exporter. Only certain number of spans getting exported to Collector agent. Below are the telemetry metrics values,

otel_scope_name : io.opentelemetry.exporters.otlp-grpc-okhttp
type: span
Metric name : otlp_exporter_seen_total : 600,000
Metric name  otlp_exporter_exported_total success: false : 400,000
Metric name  otlp_exporter_exported_total success: true : 150,000

Java agent uses default exporter : gRPC with http. Can someone help me out with understanding of the above metrics? I am trying to enable debug log in the java agent to get more insights on the error and will keep posted here.

Thanks.

**Comment by m-aqsam** [comment]
Try Out These one :

Enable debug logs in the Java agent for detailed insights.
Check network issues, collector, and exporter configurations.
Focus on resolving export failures (success: false) to enhance Reliability.
